src/openrct2/interface/Window.cpp

```diff
@@ -2148,3 +2148,18 @@ void WindowBase::ResizeFrameWithPage()
     widgets[3].right = width - 1;
     widgets[3].bottom = height - 1;
 }
+
+void WindowBase::ResizeDropdown(WidgetIndex widgetIndex, const ScreenCoordsXY& origin, int32_t widget_width)
```

Comment:
```suggestion
void WindowBase::ResizeDropdown(WidgetIndex widgetIndex, const ScreenCoordsXY& origin, int32_t width, int32_t height)
```
As I stated in the other PR this is doesn't make sense to only have one dimension.

Reply:
Alternatively, if it's only the width that changes, a member function named something like `ResizeDropdownHorizontal` would work too.

Reply:
Added the comment

Reply:
I decided to go with passing a ScreenSize object.